### PR TITLE
chore: rename pre mine

### PR DESCRIPTION
**Pull request by SWvheerden**
Description
---
renames faucet to pre-mine
Change script to add extra backup claim

### CODE REVIEW

File: base_layer/core/src/consensus/emission.rs [hunk 1]
```diff
@@ -35,7 +35,7 @@ pub struct EmissionSchedule {
     decay: &'static [u64],
     inflation_bips: u64,           // Tail inflation in basis points. 100 bips = 1 percentage point
     epoch_length: u64,             // The number of blocks in an inflation epoch
-    initial_supply: MicroMinotari, // The supply at block 0, from faucets or premine
+    initial_supply: MicroMinotari, // The supply at block 0, from premine
```
Comment (hansieodendaal) [inline base_layer/core/src/consensus/emission.rs]:
```suggestion
    initial_supply: MicroMinotari, // The supply at block 0, from pre-mine
```

File: base_layer/wallet/src/output_manager_service/service.rs [hunk 2]
```diff
@@ -1213,15 +1213,16 @@ where
...
-            if let [Opcode::CheckMultiSigVerifyAggregatePubKey(_n, _m, keys, _msg)] = output.script.as_slice() {
-                keys.clone()
-            } else {
-                return Err(OutputManagerError::ServiceError(format!(
-                    "Invalid script (TxId: {})",
-                    tx_id
-                )));
-            };
+        let public_keys = if let Some(Opcode::CheckMultiSigVerifyAggregatePubKey(_n, _m, keys, _msg)) =
+            output.script.as_slice().get(3)
```
Comment (hansieodendaal) [inline base_layer/wallet/src/output_manager_service/service.rs:1217]: Note: To conform to the new script

**Review by hansieodendaal — DISMISSED** [review]
utACK

File: base_layer/wallet/src/output_manager_service/service.rs (hunk 2 above)
Comment (SWvheerden) [reply base_layer/wallet/src/output_manager_service/service.rs:1217]: it does, node the get(3) at the end

Comment (hansieodendaal) [reply base_layer/wallet/src/output_manager_service/service.rs:1217]: Exactly :-)

**Review by hansieodendaal — APPROVED** [review]
utACK

(Please check other uses of `premine` instead of `pre-mine`)

**Pull request merged**